### DISCUSSION TITLE
角色扮演特性

### DISCUSSION THREAD

**Discussion by zhurou603** [QwenLM/Qwen3#995] [General]
有响应的文档阐述如何编写system prompt激发qwen2.5的角色扮演能力吗？
![image](https://github.com/user-attachments/assets/926ee2f8-c2e4-4c3f-91c1-51ccfca14e05)

**Comment by bigcat001** [comment]
看技术报告数据合成部分，有一篇怎么构建角色扮演数据的论文，可以看看有没有用：Large Language Models are Superpositions of All Characters: Attaining Arbitrary Role-play via Self-Alignment
